python/cuml/kernel_ridge/kernel_ridge.pyx

```diff
@@ -38,6 +38,10 @@ def _safe_solve(K, y):
         err_mode = geterr()["linalg"]
         seterr(linalg="raise")
         dual_coef = lapack.posv(K, y)
+        # Perform following check as a workaround for cusolver issue to be
+        # fixed in a future CUDA version
```

Comment:
Is there a way to identify whether this has been fixed?

Reply:
Addressed offline.